app/src/main/res/layout/activity_login.xml

```diff
@@ -124,116 +126,118 @@
     <Button
         android:id="@+id/btn_login"
         android:layout_width="match_parent"
-        android:layout_height="49dp"
-        app:layout_constraintTop_toBottomOf="@id/txt_forget"
+        android:layout_height="@dimen/main_activity_height_input_btn"
+        android:layout_marginTop="@dimen/main_activity_margin_top_btn"
+        android:background="@drawable/button_principal"
+        android:text="@string/login"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        android:layout_marginTop="19dp"
-        android:text="@string/login"
-        android:background="@drawable/button_principal"
-        />
+        app:layout_constraintTop_toBottomOf="@id/txt_forget"
+        tools:ignore="TextContrastCheck" />
+
     <TextView
         android:id="@+id/txt_opcions"
-        android:layout_width="150dp"
-        android:layout_height="35dp"
-        app:layout_constraintTop_toBottomOf="@id/btn_login"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintEnd_toEndOf="parent"
-        android:layout_marginTop="42dp"
-        android:text="@string/option_login"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/main_margin_top_options_login"
```

Comment:
remember not to tie dimens to specific screens when they can be reusable

Reply:
i'll implement it, thanks